### ISSUE TITLE
Should getent show the Display name?

### ISSUE THREAD

**Issue by vDorst** [kanidm/kanidm#385]
I wonder if `getent` should show the users Full name/Display Name?

Or did I made a configure error.

**Comment by Firstyear** [comment]
Looking at the code, the gecos field already uses the displayname attribute which should be displayed in getent?

**Comment by vDorst** [comment]
I think there is an inconsistency?

```
# kanidm account list -D idm_admin

class: account
class: memberof
class: object
class: posixaccount
displayname: René van Dorst
gidnumber: <number>
memberof: radius_access_allowed@<fqdn>
memberof: posix_group@<FQDN>
name: rene
primary_credential: primary
spn: rene@<FQDN>
unix_password: unix
uuid: <UUID>
```

`displayname` shows my full name.

But in `getent` it does not.
```
# getent passwd rene
rene:x:<number>:<number>:rene:/home/rene:/bin/fish
```

Also self info, `display` seems to be equal to `name`.
```
# kanidm self whoami -D rene
name: rene
spn: rene@<FQDN>
display: rene
uuid: 77c184c6-4b6f-4864-b363-cf2e7f78e715
groups: [Group { name: "posix_group", uuid: "<snip>" }, Group { name: "radius_access_allowed", uuid: "<snip>" }, Group { name: "rene", uuid: "<snip>" }]
claims: []
```

So should `display` equal to `name` or `displayname`?
Or is that configurable/can the user set that change that?

**Comment by Firstyear** [comment]
> I think there is an inconsistency?
> 
> ```
> # kanidm account list -D idm_admin
> 
> class: account
> class: memberof
> class: object
> class: posixaccount
> displayname: René van Dorst
> gidnumber: <number>
> memberof: radius_access_allowed@<fqdn>
> memberof: posix_group@<FQDN>
> name: rene
> primary_credential: primary
> spn: rene@<FQDN>
> unix_password: unix
> uuid: <UUID>
> ```
> 
> `displayname` shows my full name.
> 
> But in `getent` it does not.
> 
> ```
> # getent passwd rene
> rene:x:<number>:<number>:rene:/home/rene:/bin/fish
> ```

Found the bug, PR will be sent shortly. 

> 
> Also self info, `display` seems to be equal to `name`.
> 
> ```
> # kanidm self whoami -D rene
> name: rene
> spn: rene@<FQDN>
> display: rene
> uuid: 77c184c6-4b6f-4864-b363-cf2e7f78e715
> groups: [Group { name: "posix_group", uuid: "<snip>" }, Group { name: "radius_access_allowed", uuid: "<snip>" }, Group { name: "rene", uuid: "<snip>" }]
> claims: []
> ```

This one is a bug that's already fixed in 4ba9508a .

> 
> So should `display` equal to `name` or `displayname`?
> Or is that configurable/can the user set that change that?
> 
> Also XFCE shows only my `name`.

**Comment by vDorst** [comment]
> > `displayname` shows my full name.
> > But in `getent` it does not.
> > ```
> > # getent passwd rene
> > rene:x:<number>:<number>:rene:/home/rene:/bin/fish
> > ```
> 
> Found the bug, PR will be sent shortly.

Thanks, now it works fine.

```
# getent passwd rene
rene:x:<number>:<number>:René van Dorst:/home/rene:/bin/fish
```
 
> > Also self info, `display` seems to be equal to `name`.
> > ```
> > # kanidm self whoami -D rene
> > name: rene
> > spn: rene@<FQDN>
> > display: rene
> > uuid: 77c184c6-4b6f-4864-b363-cf2e7f78e715
> > groups: [Group { name: "posix_group", uuid: "<snip>" }, Group { name: "radius_access_allowed", uuid: "<snip>" }, Group { name: "rene", uuid: "<snip>" }]
> > claims: []
> > ```
> 
> This one is a bug that's already fixed in [4ba9508](https://github.com/kanidm/kanidm/commit/4ba9508a31cb9eb6976de31d1da55571f37615e9) .

Now it is not clear for me, if it is good or it is now caused by issue above?

Looking in branch history, all the branches that I currently tested, should have that commit https://github.com/kanidm/kanidm/commit/4ba9508a31cb9eb6976de31d1da55571f37615e9.

```
# git branch --contains 4ba9508a
* (HEAD detached at origin/385-displayname)
  356-chained-certs
  357-webauthn-mfa-support
  fix-356
  master
```

Currently I still see `name` with `kanidm self whoami -D rene`.
Maybe I missed updating one of kanidm programs on my system.

So can you help me out?

**Comment by Firstyear** [comment]
I'll check it today :)
